css-layout-classes/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="styles.css">
+
+  <title> CSS LAYOUT CLASSES </title>
+</head>
+<body>
+
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
```

Comment:
only thing to add would be to add this class to the CSS and set it to `width: 100%`